sormas-ui/src/main/java/de/symeda/sormas/ui/MainScreen.java

```diff
@@ -386,19 +390,24 @@ public boolean beforeViewChange(ViewChangeEvent event) {
 			if (event.getViewName().isEmpty()) {
 				// redirect to default view
 				String defaultView;
-				if (permitted(FeatureType.CASE_SURVEILANCE, UserRight.DASHBOARD_SURVEILLANCE_ACCESS)) {
-					defaultView = SurveillanceDashboardView.VIEW_NAME;
-				} else if (permitted(FeatureType.CONTACT_TRACING, UserRight.DASHBOARD_CONTACT_ACCESS)) {
-					defaultView = ContactsDashboardView.VIEW_NAME;
-				} else if (permitted(FeatureType.CAMPAIGNS, UserRight.DASHBOARD_CAMPAIGNS_ACCESS)) {
-					defaultView = CampaignDashboardView.VIEW_NAME;
-				} else if (UserProvider.getCurrent().hasUserRole(UserRole.EXTERNAL_LAB_USER)) {
-					defaultView = SamplesView.VIEW_NAME;
-				} else if (permitted(FeatureType.TASK_MANAGEMENT, UserRight.TASK_VIEW)) {
-					defaultView = TasksView.VIEW_NAME;
+				if (permitted(FeatureType.DASHBOARD, null)) {
+					if (permitted(FeatureType.CASE_SURVEILANCE, UserRight.DASHBOARD_SURVEILLANCE_ACCESS)) {
+						defaultView = SurveillanceDashboardView.VIEW_NAME;
+					} else if (permitted(FeatureType.CONTACT_TRACING, UserRight.DASHBOARD_CONTACT_ACCESS)) {
+						defaultView = ContactsDashboardView.VIEW_NAME;
+					} else if (permitted(FeatureType.CAMPAIGNS, UserRight.DASHBOARD_CAMPAIGNS_ACCESS)) {
+						defaultView = CampaignDashboardView.VIEW_NAME;
+					} else if (UserProvider.getCurrent().hasUserRole(UserRole.EXTERNAL_LAB_USER)) {
+						defaultView = SamplesView.VIEW_NAME;
```

Comment:
Hiding the sample, task or about view behind whether or not dashboards are enabled doesn't make much sense; please move those out of the parent if clause and remove the additional else clause that redirects to the cases view (because the cases view can be disabled for certain users as well, so the about view is the better default).